components/automate-gateway/api/iam/v2/request/tokens.proto

```diff
@@ -6,22 +6,33 @@ option go_package = "github.com/chef/automate/components/automate-gateway/api/ia
 import "google/protobuf/wrappers.proto";
 
 message CreateTokenReq {
+    // Unique, user-specified ID. Cannot be changed.
     string id = 1;
+    // User-specified name.
     string name = 2;
+    // Active state. Defaults to true.
+    // If set to false, token will not be authenticated or authorized.
     google.protobuf.BoolValue active = 3;
+    // Unique, optionally user-specified value.
     string value = 4;
+    // Array of projects the token is in. Empty by default.
     repeated string projects = 5;
 }
 
+
 message GetTokenReq {
     string id = 1;
 }
 
 message UpdateTokenReq {
-    // ID can't be changed; ID used to discover token
+    // Unique, user-specified ID. Cannot be changed.
```

Comment:
Wish there was a way to specify this as a path parameter only so it didn't show up twice 🤔 But that might be Next Level open API